src/containers/releaseNotes/CategoryReleaseNotesPage.tsx

```diff
@@ -1,84 +1,57 @@
-    const { apis } = apiDefs[apiCategoryKey];
+import CategoryReleaseNotesCardSection from './CategoryReleaseNotesCardSection';
+import CategoryReleaseNotesList from './CategoryReleaseNotesList';
 
-    let cardSection;
-    if (apis.length > 1) {
-      const apiCards = apis.map((apiDesc: IApiDescription) => {
-        const { description, name, urlFragment, vaInternalOnly, trustedPartnerOnly } = apiDesc;
-        const dashUrlFragment = urlFragment.replace('_', '-');
-
-        return (
-          <Flag key={name} name={`hosted_apis.${urlFragment}`}>
-            <CardLink
-              name={name}
-              subhead={
-                vaInternalOnly || trustedPartnerOnly ? (
-                  <OnlyTags {...{ vaInternalOnly, trustedPartnerOnly }} />
-                ) : (
-                  undefined
-                )
-              }
-              url={`/release-notes/${apiCategoryKey}#${dashUrlFragment}`}
-            >
-              {description}
-            </CardLink>
-          </Flag>
-        );
-      });
-
-      cardSection = (
-        <div role="navigation" aria-labelledby={`${apiCategoryKey}-release-notes`}>
-          <div className={defaultFlexContainer()}>{apiCards}</div>
-        </div>
-      );
-    }
+const CategoryReleaseNotesPageContent = ({ apiCategory, apiFlagName } : { apiCategory: BaseAPICategory, apiFlagName: string }) => {
 
+  if (apiCategory && apiFlagName !== '') {
     return (
-      <section role="region" aria-labelledby={`${apiCategoryKey}-release-notes`}>
+      <section role="region" aria-labelledby={`${apiCategory.name}-release-notes`}>
         <PageHeader
-          halo={apiDefs[apiCategoryKey].name}
+          halo={apiCategory.name}
           header="Release Notes"
-          id={`${apiCategoryKey}-release-notes`}
+          id={`${apiCategory.name}-release-notes`}
+        />
+        <CategoryReleaseNotesCardSection
+          apiCategory={apiCategory}
+          apiFlagName={apiFlagName}
+        />
+        <CategoryReleaseNotesList
+          apiCategory={apiCategory}
+          apiFlagName={apiFlagName}
         />
-        {cardSection}
-        <div className={classNames('vads-u-width--full', 'vads-u-margin-top--4')}>
-          {apis.map(api => (
-            <ApiReleaseNote key={api.urlFragment} api={api} />
-          ))}
-        </div>
       </section>
     );
   }
-}
+  return null;
+};
+
+export const ActiveCategoryReleaseNotesPage = (props: RouteComponentProps<IApiNameParam>) => {
+  const { apiCategoryKey } = props.match.params;
+  const categoryDefinition = getApiDefinitions()[apiCategoryKey];
+  return (
+    <CategoryReleaseNotesPageContent
+      apiCategory={categoryDefinition}
+      apiFlagName='hosted_apis'
+    />
+  );
+};
+
+export const DeactivatedReleaseNotesPage = (props: RouteComponentProps<IApiNameParam>) => {
+  const apiCategoryKey = 'deactivated';
+  const categoryDefinition = getDeactivatedCategory()[apiCategoryKey];
```

Comment:
```suggestion
  const categoryDefinition = getDeactivatedCategory();
```

rewrite `getDeactivatedCategory()` to return the "category" definition. "deactivated" isn't really a category key, and it's not clear what other keys would go in that returned object in the future (plus, not future-proofing can be a good guideline sometimes).